x-pack/plugins/ml/public/ui_actions/types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  HasType,
+  PublishesLocalUnifiedSearch,
+  PublishesPanelTitle,
+  PublishesSavedObjectId,
+} from '@kbn/presentation-publishing';
+import type { LensApi } from '@kbn/lens-plugin/public';
+import type { MapApi } from '@kbn/maps-plugin/public';
+
+export type DashboardApi = Partial<
```

Comment:
`Dashboard` is in the `requiredBundles` of ML. I think this means we can import the `DashboardApi` type directly from the plugin there rather than duplicating it here.